pkgs/test/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 ## 1.19.0-dev
 
-* Support query parameters `name` and `full-name` on test paths, which will
-  apply the filters to only those test suites.
+* Support query parameters `name`, `full-name`, `line`, and `col` on test paths,
+  which will apply the filters to only those test suites.
   * All specified filters must match for a test to run.
   * Global filters (ie: `--name`) are also still respected and must match.
+  * The `line` and `col` will match if any frame from the test trace matches
+    (the test trace is the current stack trace where `test` is invoked).
+    * If either `line` or `col` are used, stack trace chaining is enabled for
```

Comment:
Does this mainly matter for web tests? I can't find where in the code this happens - can you point me to it?

Reply:
It happens for all tests that run remotely (so for everything but the direct dart file invocation case), specifically here https://github.com/dart-lang/test/pull/1604/files#diff-d694d0b45775f9db69469d5db79e8b6f0bbb05b8ddcf110340a1e205c454f5a6R190

Reply:
Ah, that's a different behavior. That isn't related to `--chain-stack-traces` or `Chain.capture`.

I think we can skip this mention from the user facing changelog - it's a minor detail of the communication between the runner and the tests, but it doesn't impact user visible behavior and shouldn't impact perf as much as stack trace chaining would.

Reply:
Ah ok, ya I was a bit confused by it